pkg/kubelet/dockershim/network/plugins.go

```diff
@@ -247,18 +248,41 @@ func getOnePodIP(execer utilexec.Interface, nsenterPath, netnsPath, interfaceNam
 	return ip, nil
 }
 
+// preferredFamily will determine the preferred IP family, based on the DNS service IP
+// environment variable (DNS_SVC_IP), if set.
+func preferredFamily() string {
+
+	dnsServiceIP := os.Getenv("DNS_SVC_IP")
+	if dnsServiceIP == "" {
+		glog.V(3).Infof("Unable to determine preferred IP family")
+		return ""
+	}
+	glog.V(3).Infof("DNS Service IP is %s", dnsServiceIP)
+	dnsIP := net.ParseIP(dnsServiceIP)
+	if dnsIP == nil {
+		glog.Warningf("Unable to parse DNS_SVC_IP (%s) to determine preferred family", dnsServiceIP)
```

Comment:
i think it make sense to indicate that the fall back here is to -4.

simplest way is to use a `goto` to a label before:
```
fallback:
	glog.V(3).Infof("Using IPv4.....
```
yet i see that `goto` is not that widely used in the project.

Reply:
I'm changing the wording of the message. Let me know if that is OK.